### DISCUSSION TITLE
[Feature Request] live search pagination

### DISCUSSION THREAD

**Discussion by mattstein** [craftcms/docs#303] [Ideas]
### Description

Increase the suggestion limit and paginate beyond ten results.

**Comment by domstubbs** [comment]
I find myself constantly struggling to locate fairly basic stuff in the docs and I wonder whether this 10 result limit is the main culprit.

Case in point, a class reference search for “request” only includes results for `craft-console-request`. The result that most people will be after, `craft-web-request`, is nowhere to be found.

I’m not familiar with Algolia, but if exact class matches outranked partial method matches that would probably help matters too.

**Comment by mattstein** [comment]
Thanks @domstubbs. I think what you’re describing is more a problem with relevance than the size of the result set. 

We switched the [product documentation](https://craftcms.com/docs) to use [FlexSearch](https://github.com/nextapps-de/flexsearch) for a bit more control over the indexes and results, but the [class references](https://docs.craftcms.com/api/v3/) still use Algolia so improvement there would have to be in the [docsearch config](https://github.com/algolia/docsearch-configs/blob/master/configs/craftcms.json) or the [standalone VuePress theme](https://github.com/pixelandtonic/vuepress-theme-craftdocs) but not this project. 

I appreciate that example though, and don’t hesitate to share others in GitHub issues or in the #docs Discord channnel!
